src/us/kbase/workspace/test/kbase/JSONRPCLayerTest.java

```diff
@@ -904,7 +904,9 @@ public void saveAndGetObjects() throws Exception {
 				.withName("auto1");
 		Map<String, List<String>> exp = new HashMap<String, List<String>>();
 		ObjectData objo = CLIENT1.getObjects2(new GetObjects2Params()
-			.withObjects(toObjSpec(Arrays.asList(ojbid)))).getData().get(0);
+			.withObjects(toObjSpec(Arrays.asList(ojbid)))
+			.withSkipExternalAclUpdates(1L) // should have no effect
+			).getData().get(0);
```

Comment:
is there another test where you verify that adding the skip external ACL updates param has no effect, either implicitly or explicitly?

Reply:
there's not really another way to test this, I think. The current tests are:

* when no external systems are connected to the workspace, the new param does nothing. In that case, there's nothing to query to ensure nothing happened (this is the test above).
* when external systems are connected (in the handle/shock test and the samples test), getting objects is tested with and without the new parameter to ensure it works as expected and the ACLs are updated or not.

I'm not seeing another way to test off the top of my head, anyway